lib/connection.ts

```diff
@@ -374,7 +374,14 @@ export class Connection {
             console.warn(
               `Received event for unknown subscription ${message.id}. Unsubscribing.`,
             );
-            this.sendMessagePromise(messages.unsubscribeEvents(message.id));
+            this.sendMessagePromise(
+              messages.unsubscribeEvents(message.id),
+            ).catch((err) => {
+              console.warn(
+                `Error unsubsribing from unknown subscription ${message.id}`,
+                err,
+              );
```

Comment:
No need to throw another warning. The goal (no subscription for message.id) has been achieved. 
```suggestion
```

Reply:
How about if I wrap it in an `if (DEBUG)` ?
Upon catching the error, we don't know for sure the goal has been achieved - it depends on what the error is.
Or I can just delete the `console.warn` as you suggest.

Reply:
I wrapped the warning in an if (DEBUG).  Alternatively, can remove and turn into a comment if you want.